### PR TITLE
138672663 add contributing markdown v5

### PR DESCRIPTION
**Pull request by tmikeschu**
Finish contributing markdown

Add instructions for setting up environment and finding API token.

[#138672663]

### CODE REVIEW

**Comment by kheppenstall** [comment]
Nice work here @tmikeschu. Looks good for me.

File: Contributing.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+To get your environment setup for contribution:
+
+1. Prepare your environment variables for authentication in your API requests.
+
+```
+cp .env.example .env
+```
+
+2. Replace the values of the `.env` variables with your PivotalTracker credentials.
```
Comment (jasonnoble) [inline Contributing.md]: `Replace the variables in the `.env` file with your ...`

File: Contributing.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+PIVOTAL_TOKEN_ID='98sf9839848fasd989328asc'
+PIVOTAL_EMAIL='your@email.here'
+PIVOTAL_USER_NAME='yourusername'
+PIVOTAL_USER_PASSWORD='yourpassword'
+```
+
+### To find your API token:
+   1. Click your username dropdown on the top right of the screen (https://www.pivotaltracker.com/dashboard).
+   2. Click 'Profile'
+   3. Your API token is at the bottom of the list. Copy it for your `.env` file.
```
Comment (jasonnoble) [inline Contributing.md:34]: Make sure your `.env` file is added to your global gitignore file (How-To:  https://help.github.com/articles/ignoring-files/#create-a-global-gitignore)

File: Contributing.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+```
+
+2. Replace the values of the `.env` variables with your PivotalTracker credentials.
+
+```
+PIVOTAL_TOKEN_ID='98sf9839848fasd989328asc'
+PIVOTAL_EMAIL='your@email.here'
+PIVOTAL_USER_NAME='yourusername'
+PIVOTAL_USER_PASSWORD='yourpassword'
+```
```
Comment (jasonnoble) [inline Contributing.md]: What about `* Create a Pivotal Tracker project and modify your .env file to have the project ID` ?

File: Contributing.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+1. Prepare your environment variables for authentication in your API requests.
+
+```
+cp .env.example .env
+```
+
+2. Replace the values of the `.env` variables with your PivotalTracker credentials.
+
+```
+PIVOTAL_TOKEN_ID='98sf9839848fasd989328asc'
```
Comment (jasonnoble) [inline Contributing.md]: `PIVOTAL_TOKEN_ID='98sf98_YOUR_ACTUAL_TOKEN_328asc'`

**Review by jasonnoble — CHANGES_REQUESTED** [review]

File: Contributing.md (hunk 3 above)
Comment (tmikeschu) [reply Contributing.md]: Thanks!

File: Contributing.md (hunk 2 above)
Comment (tmikeschu) [reply Contributing.md:34]: 👍 linked

File: Contributing.md (hunk 4 above)
Comment (tmikeschu) [reply Contributing.md]: Roger that.

File: Contributing.md (hunk 1 above)
Comment (tmikeschu) [reply Contributing.md]: 👍

**Pull request merged**